.changeset/tame-pigs-appear.md

```diff
@@ -0,0 +1,5 @@
+---
+'@primer/react-brand': patch
+---
+
+Adds spacing scale support in Stack's gap and padding props
```

Comment:
Is it possible to add some code examples for the changelog please?

Something like:

```jsx
<>
<Stack gap={96} padding={96}>...</Stack>
<Stack 
  gap={{
    narrow: 96
  }} 
  padding={{
    narrow: 96
   }}>
  ...
</Stack>
</>
```